docs/user-guide/kubectl-overview.md

```diff
@@ -135,6 +135,66 @@ For explanation, here I gave some mostly often used `kubectl` command examples.
 		// Starts streaming of logs from pod <pod-name>, it is something like 'tail -f'.
 		$ kubectl logs -f <pod-name>
 
+                // Get the logs from a previous run of a particular pod
+                $ kubectl logs --previous <pod-name>
+
+## Kubectl output
+
+`kubectl` supports a variety of output formats for different use cases.  The output format is selected with the `-o/--output` flag.
+
+The following options are supported:
+
+Output Format | Description
+--------------| -----------
+<default>     | Human readable format
+`-o=wide`     | Human readable format but with more columns
+`-o=yaml`     | YAML formatted API object
+`-o=json`     | JSON formatted API object
+`-o=name`     | Print the resource name and nothing else
+`-o=jsonpath=<template>` | Print fields selected by a [jsonpath](jsonpath.md) expression.
+`-o=jsonpath-file=<filename>` | Print the fields specified by [jsonpath](jsonpath.md) expressions in <filename>.
+`-o=custom-columns=<spec>` | Print a table using a comma separated list of [custom columns](#custom-columns)
+`-o=custom-columns-file=<filename>` | Print a table using a [custom columns](#custom-columns) template.
+
+### Sorting list objects
+
+Users can sort list objects (e.g. `PodList`) by any numeric or string field using the `--sort-by` flag.  The field is selected using a
+[jsonpath](jsonpath.md) expression.
+
+For example:
```

Comment:
Revise to:


"For example, to print a list of your pods that's sorted by name, you run:"